ee/tables/windowsupdatetable/windowsupdate.go

```diff
@@ -105,38 +107,12 @@ func (t *Table) generateWithLauncherExec(ctx context.Context, queryContext table
 
 	var results []map[string]string
 
-	for _, locale := range tablehelpers.GetConstraints(queryContext, "locale", tablehelpers.WithDefaults("_default")) {
-		args := []string{
-			"query-windowsupdates",
-			"-locale", locale,
-			"-table_mode", strconv.Itoa(int(t.mode)),
-		}
-		cmd := exec.CommandContext(ctx, launcherPath, args...) //nolint:forbidigo // We can exec the current executable safely
-		cmd.Env = append(cmd.Env, "LAUNCHER_SKIP_UPDATES=TRUE")
-		out, err := cmd.CombinedOutput()
+	for _, locale := range tablehelpers.GetConstraints(queryContext, "locale", tablehelpers.WithDefaults(defaultLocale)) {
```

Comment:
hrm. We're not really doing the right caching thing for the different locales. But I think to solve that we'd need to introspect the

Reply:
I went the lazy route because we appear to always use this default at the moment (no queries specifying `locale` that I could see, anyway) -- if that's subject to change, let me know?

Reply:
I am unlikely to remember. But presumably we'll hear about the performance